contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,136 +1,42 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for spectral peaks, their functional form, and support (values).
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   n_support: |
     Number of support points
 type: group
 NXpeak(NXobject):
-  label:
+  label(NX_CHAR):
     doc: |
-      Human-readable identifier to specify which concept/entity
+      Human-readable label which specifies which concept/entity
       the peak represents/identifies.
-  peak_model:
+  description(NX_CHAR):
     doc: |
       Is the peak described analytically via a functional form
       or is it empirically defined via measured/reported
       intensity/counts as a function of an independent variable.
       
-      If the functional form is not empirical or gaussian, users
+      If the functional form is not empirical or Gaussians, users
       should enter other for the peak_model and add relevant details
       in the NXcollection.
-    enumeration: [empirical, gaussian, lorentzian, other]
+    enumeration: [empirical, gaussian, lorentzian, voigt, other]
   position(NX_NUMBER):
-    unit: NX_ANY
     doc: |
       In the case of an empirical description of the peak and its shoulders,
       this array holds the position values for the independent variable.
-    dimensions:
-      rank: 1
-      dim: [[1, n_support]]
-  intensity(NX_NUMBER):
     unit: NX_ANY
+    dim: (n_support,)
+  intensity(NX_NUMBER):
     doc: |
       In the case of an empirical description of the peak and its shoulders,
       this array holds the intensity/count values at each position.
-    dimensions:
-      rank: 1
-      dim: [[1, n_support]]
+    unit: NX_ANY
+    dim: (n_support,)
   (NXcollection):
```

Comment:
Why not use `NXparameters` here?

I plan to extend `NXpeak` in #170, maybe we can also talk about that there.

Reply:
I second this proposal, further discussion should be done in #170